gallery/src/data-visualization/ReactVisPage.tsx

```diff
@@ -0,0 +1,150 @@
+      <p className="nx-p">
+        <code className="nx-code">{'<XYPlot height={300} width = {300} />'}</code>
+      </p>
+      <h3 className="nx-h3">Ordering</h3>
+      <p className="nx-p">
+        Similar to traditional React, order matters as components are drawn in order. For example, placing
+        <code className="nx-code">{'<VerticalGridLines />'}</code> before
+        <code className="nx-code">{'<LineSeries data={data} />'}</code> will draw the chart on top of the gridlines.
+      </p>
+      <h3 className="nx-h3">Colors</h3>
+      <p className="nx-p">
+        Colors are assigned to charts by passing in the <code className="nx-code">color</code> or the
+        <code className="nx-code">stroke</code> prop depending on the type of chart being used.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<LineSeries />'}</code> charts use the <code className="nx-code">stroke</code>
+        {' '} prop to determine colors of the lines.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<BarSeries />'}</code> charts use the <code className="nx-code">color</code>
+        {' '} prop to determine colors of the bars.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<RadialChart />'}</code> uses the <code className="nx-code">color</code>
+        {' '} property on the <code className="nx-code">data</code> object to determine colors of the bars.
+      </p>
+      <p className="nx-p">
+        RSC's <code className="nx-code">{'selectableColors'}</code> array provides a seamlessly way of
+        assigning colors based on RSC color guidelines. Usage is quite simple. Simply import the
+        <code className="nx-code">{'selectableColors'}</code> array and pass it on the
+        <code className="nx-code">stroke</code> or the <code className="nx-code">color</code> prop. An example
+        of using the color purple (<code className="nx-code">selectableColors[0]</code>) is shown below:
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<LineSeries data={chartData} stroke={selectableColors[0]}/>'}</code>
+      </p>
+      <NxWarningAlert>
+        When assigning colors to charts, it is critical to choose colors from RSC's
+        {' '} <code className="nx-code">{'selectableColors'}</code> array.
+        For more details on colors, please check the Color Palettes and Selectable Colors section.
+      </NxWarningAlert>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <p className="nx-p">
+          Note: The following table only describles the props used for examples on this page.
+          For the full official API documentation, please click {' '}
+          <a href="https://uber.github.io/react-vis/documentation/welcome-to-react-vis" target="_blank">here.</a>
+        </p>
+        <NxTable>
```

Comment:
It's a bit confusing having this single table that appears to combine docs for props on a number of different components.

Reply:
> It's a bit confusing having this single table that appears to combine docs for props on a number of different components.

I initially had a single page that contained both the docs and the examples, but @glenhunter and I talked about it and he suggested that we break it into a "main page" that has descriptions of the common props and then break out each example graph into its own page and have the prop descriptions for props that are _only_ used in that specific example.

Would it be better if each graph example page had descriptions of _all_ of the props used in that example? And perhaps remove the documentation table from the main page entirely?